js_tests/admin/SelectBox.test.js

```diff
@@ -8,7 +8,7 @@ QUnit.test('init: no options', function(assert) {
     var $ = django.jQuery;
     $('<select id="id"></select>').appendTo('#qunit-fixture');
     SelectBox.init('id');
-    assert.equal(SelectBox.cache.id.length, 0);
+    assert.equal(SelectBox.cache.id.length, 1);
```

Comment:
At a glance I'm unsure why this should change...

Reply:
when console SelectBox.cache.id.length i got 1 so , I'll update with 1,
anyway I'll fix that 
thanks